### ISSUE TITLE
If you have some issues new version

### ISSUE THREAD

**Issue by Davlatmurod** [philippfrenzel/yii2fullcalendar#146]
Thanks new version. But I am tried new version issues. 
I recommend last version add to composer.json file and update.
"philippfrenzel/yii2fullcalendar": "3.9.0"

**Comment by f1dz** [comment]
Yeah, I prefer to use this version. The new version has so many issues need to be fixed.

**Comment by sheershoff** [comment]
The correct command to downgrade if requiring blames lock file that it is impossible to downgrade:

`composer require --verbose --prefer-dist --update-with-dependencies philippfrenzel/yii2fullcalendar:3.9.0`
